packages/grid/_modules_/grid/hooks/root/useGridContainerProps.ts

```diff
@@ -71,6 +71,10 @@ export const useGridContainerProps = (
 
       scrollBarSize.y = hasScrollY ? options.scrollbarSize! : 0;
```

Comment:
This scrollbar behavior impacts both directions. Seeing x and y axis handled differently seems off.